Source/dom/index.js

```diff
@@ -1,46 +1,82 @@
-import { AnimationType, BackTarget } from './models/Flow'
-import './models/DataOverride'
-
-export {
-  exportObject as export,
-  objectFromJSON as fromSketchJSON,
-} from './export'
-
-export { createLayerFromData } from './import'
-
-export { Document, getDocuments, getSelectedDocument } from './models/Document'
-export { Library, getLibraries } from './models/Library'
-export { SharedStyle } from './models/SharedStyle'
-export { Rectangle } from './models/Rectangle'
-
-export { Style } from './style/Style'
-
-export { Group } from './layers/Group'
-export { Text } from './layers/Text'
-export { Image } from './layers/Image'
-export { Shape } from './layers/Shape'
-export { ShapePath } from './layers/ShapePath'
-export { Artboard } from './layers/Artboard'
-export { Page } from './layers/Page'
-export { SymbolMaster } from './layers/SymbolMaster'
-export { SymbolInstance } from './layers/SymbolInstance'
-export { HotSpot } from './layers/HotSpot'
-export { Slice } from './layers/Slice'
-
-export { Types } from './enums'
-export { wrapObject as fromNative } from './wrapNativeObject'
-export {
+const { AnimationType, BackTarget } = require('./models/Flow')
+require('./models/DataOverride')
+
+const { exportObject, objectFromJSON } = require('./export')
+
+const { createLayerFromData } = require('./import')
+
+const {
+  Document,
+  getDocuments,
+  getSelectedDocument,
+} = require('./models/Document')
+const { Library, getLibraries } = require('./models/Library')
+const { SharedStyle } = require('./models/SharedStyle')
+const { Rectangle } = require('./models/Rectangle')
+
+const { Style } = require('./style/Style')
+
+const { Layer } = require('./layers/Layer')
```

Comment:
The previous version of this file did not export `Layer`, right? Just double checking this is intentional now.

Reply:
Ah it wasn't on purpose in this particular PR but it's a change I wanted to make so it's fine 👍 (it was probably a un-commited change before).

It allows people to extend Layer with their own functions and make every other object inherit from it.